react/features/mobile/audio-mode/components/AudioRoutePickerDialog.js

```diff
@@ -146,21 +179,66 @@ class AudioRoutePickerDialog extends Component<Props, State> {
         this._hide();
     }
 
-    _onSubmit: (?Object) => void;
+    _onSelect: (?Object) => void;
 
     /**
      * Handles the selection of a device on the sheet. The selected device will
      * be used by {@code AudioMode}.
      *
-     * @param {Object} device - Object representing the selected device.
+     * @param {Device} device - Object representing the selected device.
      * @private
      * @returns {void}
      */
-    _onSubmit(device) {
+    _onSelect(device: Device) {
         this._hide();
         AudioMode.setAudioDevice(device.type);
     }
 
+    /**
+     * Renders a single device.
+     *
+     * @param {Device} device - Object representing a single device.
+     * @private
+     * @returns {ReactElement}
+     */
+    _renderDevice(device: Device) {
+        const { iconName, selected, text } = device;
+        const selectedStyle = selected ? styles.selectedText : {};
+
+        return (
+            <TouchableHighlight
+                key = { device.type }
+
+                // TODO The following disables an eslint error alerting about a
+                // known potential/theoretical performance pernalty:
+                //
+                // A bind call or arrow function in a JSX prop will create a
+                // brand new function on every single render. This is bad for
+                // performance, as it will result in the garbage collector being
+                // invoked way more than is necessary. It may also cause
+                // unnecessary re-renders if a brand new function is passed as a
+                // prop to a component that uses reference equality check on the
+                // prop to determine if it should update.
+                //
+                // I'm not addressing the potential/theoretical performance
+                // penalty at the time of this writing because it doesn't seem
+                // to me that it's a practical performance penalty in the case.
+                //
+                // eslint-disable-next-line react/jsx-no-bind
+                onPress = { this._onSelect.bind(this, device) }
```

Comment:
This indeed doesn't seem like a huge performance issue for me either, but can you please check what we did with Lyubo in the VideoSwitch component for a similar issue? Maybe that's an easy win to remove this comment and the eslint hint.

Reply:
Can you point me to a commit I can look at?

Reply:
Ah, found it. Will fix.